content/hi/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,129 @@
+---
+# reviewers:
+# - lavalamp
+title: कुबेरनेट्स घटक
+content_type: concept
+description: >
+  कुबेरनेट्स क्लस्टर में कंट्रोल प्लेन के घटक और मशीनों का एक समूह होता है।
+  विमान और मशीनों का एक सेट होता है जिसे नोड्स कहा जाता है।
+weight: 30
+card:
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+जब आप कुबेरनेट्स को डिप्लॉय करते हैं, तो आपको एक क्लस्टर मिलता है।
+{{< glossary_definition term_id="cluster" length="all" prepend="कुबेरनेट्स क्लस्टर में">}}
+
+यह दस्तावेज़ आपके लिए आवश्यक विभिन्न घटकों की रूपरेखा देता है
+एक पूर्ण और कार्यशील कुबेरनेट क्लस्टर।
```

Comment:
यह दस्तावेज़ एक पूर्ण और कार्यशील कुबेरनेट्स क्लस्टर के लिए आवश्यक विभिन्न घटकों की रूपरेखा तैयार करता है।

At line 19 and 20  the sentence at 20 is not giving correct explaination so we can compile the line as above WDYT?